src/models/SqlOutputContentProvider.ts

```diff
@@ -294,8 +294,11 @@ export class SqlOutputContentProvider implements vscode.TextDocumentContentProvi
             queryRunner.eventEmitter.on('resultSet', (resultSet) => {
                 this._service.broadcast(resultsUri, 'resultSet', resultSet);
             });
-            queryRunner.eventEmitter.on('batch', (batch) => {
-                this._service.broadcast(resultsUri, 'batch', batch);
+            queryRunner.eventEmitter.on('batchStart', (batch) => {
```

Comment:
Perhaps make a task / issue to track this?

---
In reply to: [91421282](https://github.com/Microsoft/vscode-mssql/pull/480#discussion_r91421282) [](ancestors = 91421282)